source/_integrations/yardian.markdown

```diff
@@ -11,22 +12,45 @@ ha_codeowners:
   - '@h3l1o5'
 ha_domain: yardian
 ha_platforms:
+  - binary_sensor
+  - sensor
   - switch
 ha_integration_type: integration
 ---
 
 The **Yardian** {% term integration %} allows you to control your [Yardian Smart Sprinkler Controller](https://yardian.com/products/yardian-pro-smart-sprinkler-controller/).
 
-There is currently support for the following platform within Home Assistant:
+There is currently support for the following platforms within Home Assistant:
 
 - Switch - Allows you to view the status of zones and control them.
+- Binary sensor - Shows watering status along with standby and freeze prevent diagnostics. Per-zone enabled diagnostics are provided but disabled by default.
+- Sensor - Reports rain delay, the number of active zones, and diagnostic values such as sensor delay, water hammer duration, and the controller region. Most diagnostic sensors are disabled by default.
 
 {% include integrations/config_flow.md %}
 
 During the configuration, you will have to manually set the **Host** and the **Access Token**. You can find them inside your [Yardian App](https://yardian.com/app/).
 
 ![Yardian Host/Token Location](/images/integrations/yardian/yardian_config_flow.jpg)
 
+## Supported functionality
+
+The **Yardian** integration provides the following entities.
+
+### Binary sensors
+
+- **Watering running**: Is `on`  when a zone is currently irrigating.
+- **Standby**:  Is `on` when the controller is in standby mode.
+- **Freeze prevent**: Turns on when the controller enables freeze prevention.
+- **Zone enabled**: On if a zone is enabled. These entities are disabled by default and created per zone
+
+### Sensors
+
+- **Rain delay**: Remaining rain delay reported in seconds.
+- **Active zone count**: The number of zones currently running.
+- **Sensor delay**: Duration in seconds (disabled by default).
+- **Water hammer duration**: Duration in seconds (disabled by default).
```

Comment:
what do you mean by water hammer duration? How long it takes for the pressure surge to stop after the wtaer has been turned of?